nodeup/pkg/bootstrap/install.go

```diff
@@ -161,6 +161,7 @@ func (i *Installation) buildSystemdJob() *nodetasks.InstallService {
 	manifest := &systemd.Manifest{}
 	manifest.Set("Unit", "Description", "Run kOps bootstrap (nodeup)")
 	manifest.Set("Unit", "Documentation", "https://github.com/kubernetes/kops")
+	manifest.Set("Unit", "ConditionPathExists", "!/srv/kubernetes/kubelet-server.crt")
```

Comment:
if I understood correctly we cannot use this in AWS? Is there easy way how we could pass cloud information to this package @olemarkus